### PR TITLE
Ashwalker Lavaland Kitchen Remap

### LINKED ISSUE #14780: Zero gravity in Ashwalker's nest
So, if the station's gravity generator turns off, there is no gravity in the Ashwalker's kitchen.
image

### PR DESCRIPTION
**Pull request by Leudoberct1**
**About The Pull Request**

This reworks the kitchen in the ashwalker lavaland village to be something more thematically appropriate. The current one doesn't make much sense in the village, and it doesn't even have gravity due to some funky areas. This new kitchen is more tribalistic, and mirrors some features from the icebox village.

<details>
<summary> Before & After </summary>

Before:

image

After:

image

</details>

close: #14780

**How This Contributes To The Skyrat Roleplay Experience**

A bunch of people have complained about the kitchen for the Ashwalkers and how it's unfitting for the theme, as well as the lack of gravity inside. This just makes it something more thematically appropriate that also has gravity.

**Changelog**

:cl:
add: Remapped the Ashwalker kitchen to be more thematically appropriate.
/:cl:

### CODE REVIEW

File: html/changelogs/Leudoberct1-PR-1.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+################################
```
Comment (Zonespace27) [inline html/changelogs/Leudoberct1-PR-1.yml]: no clue how you got this here but this entire file isn't needed

Comment (Leudoberct1) [reply html/changelogs/Leudoberct1-PR-1.yml]: the changelog? i was told that changelogs are part of the PR standards here- you get this same stuff on aurora

Comment (Jolly-66) [reply html/changelogs/Leudoberct1-PR-1.yml]: The way TG compiles CL entries is AFTER the PR is merged. It doesn't need any input from you.

For reference, your PR body is what compiles the CL. You need to add that back.

**Review by Jolly-66 — CHANGES_REQUESTED** [review]
Right. Please follow the PR template. Its there for a reason.

Also per my comment in Zones comment above, we compile change logs differently here. Delete the `.yml`,

EDIT: Also, I would 100% prefer if you just dumped the map edit photos into the PR body itself instead of it being external links. Use markdowns spoilers to condense them if your PR body is too big (which is preferred).

**Review by Jolly-66 — APPROVED** [review]
Map looks fine to me, and is way better aesthetically. This was something on my "eventual to-do list", but thank you no less.

**Comment by Jolly-66** [comment]
Also, can you put `close: #14780` somewhere in your PR body? This'll help fix this issue. Ideally, you can use fixes instead.

**Comment by jjpark-kb** [comment]
If no actions are taken within two days, I will close this.

**Pull request merged**